tests/testthat/test-replace-negatives.R

```diff
@@ -0,0 +1,30 @@
+library(dplyr)
+testthat::context("replace_negatives")
+
+# read in data
+data <- covidData::load_jhu_data(spatial_resolution = "state", 
+                     temporal_resolution = "daily",
+                     measure = "deaths", replace_negatives = FALSE, 
+                     adjustment_cases = "none") %>%
+  # CO
+  dplyr:: filter(location == "08")
+
```

Comment:
Let's add a second example test case where there are not negative reports to replace.